sale_stock_picking_note/migrations/13.0.1.1.0/post-migration.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    # Changes old values to picking_comments
```

Comment:
This scripts assume that the current users are missusing the module, which would only be possible if the delivery slip printed the picking note field, which isn't the case.

If a user has such modification, this script should go to their custom module.

Reply:
Indeed. Don't touch this here.